### PR TITLE
Add uncordon step to containerd migration doc

### PR DESCRIPTION
**Pull request by mattster98**
Added a step to uncordon the node since it was drained at the beginning of this process.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 1]
```diff
@@ -129,6 +129,14 @@ systemctl start kubelet
 
 Run `kubectl get nodes -o wide` and containerd appears as the runtime for the node we just changed.
 
+## Uncordon the node 
+
+```shell
+kubectl uncordon <node-to-uncordon>
+```
+
+Replace `<node-to-uncordon>` with the name of your node you previously drained.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:138]: How about doing this as the final step, and reword “Finally if everything goes well, remove Docker.”

?

Comment (mattster98) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md]: Done

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 2]
```diff
@@ -168,3 +168,11 @@ To delete them, follow Docker's instructions to [Uninstall Docker Engine](https:
...
 Docker's instructions for uninstalling Docker Engine create a risk of deleting containerd. Be careful when executing commands.
 {{< /caution >}}
+
+## Uncordon the node 
+
+```shell
+kubectl uncordon <node-to-uncordon>
+```
+
+Replace `<node-to-uncordon>` with the name of your node you previously drained.
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:178]: nit: Replace `<node-to-uncordon>` with the name of the node you previously drained.

**Pull request merged**